common/app/model/meta.scala

```diff
@@ -59,12 +59,26 @@ object Fields {
       displayHint = apiContent.fields.flatMap(_.displayHint).getOrElse(""),
       isLive = apiContent.fields.flatMap(_.liveBloggingNow).getOrElse(false),
       sensitive = apiContent.fields.flatMap(_.sensitive),
-      shouldHideReaderRevenue = apiContent.fields.flatMap(_.shouldHideReaderRevenue),
+      shouldHideReaderRevenue = shouldHideReaderRevenue(apiContent),
       legallySensitive = apiContent.fields.flatMap(_.legallySensitive),
       firstPublicationDate = apiContent.fields.flatMap(_.firstPublicationDate).map(_.toJodaDateTime),
       lang = apiContent.fields.flatMap(_.lang)
     )
   }
+
+  private def shouldHideReaderRevenue(apiContent: contentapi.Content): Option[Boolean] = {
+    val cutoffDate = new DateTime("2017-07-03T12:00:00.000Z")
```

Comment:
@mchv @LATaylor-guardian why is this flag the negation rather than being `showReaderRevenue`? For old content this flag would be absent and therefore could be considered false, and new content will have a value set.